.github/workflows/build.yml

```diff
@@ -21,16 +21,7 @@ permissions:
   contents: write
 
 jobs:
-
-  lint:
-    uses: ./.github/workflows/lint.yml
-
-  tests:
-    uses: ./.github/workflows/tests.yml
-
   build:
-    if: ${{ (github.ref == 'refs/heads/main') || startsWith(github.ref, 'refs/tags/v') }}
-    needs: [lint, tests]
```

Comment:
lint and tests are launch anyway, let me check so stuff

Reply:
I removed 

```
lint:
    uses: ./.github/workflows/lint.yml

  tests:
    uses: ./.github/workflows/tests.yml
```

because lint and test workflow were executed 2 times: workflow themself on push and here.